master/buildbot/test/unit/changes/test_svnpoller.py

```diff
@@ -23,8 +23,10 @@
 from buildbot.changes import svnpoller
 from buildbot.process.properties import Interpolate
 from buildbot.test.util import changesource
-from buildbot.test.util import gpo
```

Comment:
should we eventually remove gpo?

Reply:
Yes, there are a couple of remaining users, these will go in another PR.